flask_rebar/request_utils.py

```diff
@@ -59,8 +65,17 @@ def response(data, status_code=200, headers=None):
     :rtype: flask.Response
     """
     resp = jsonify(data)
+
+    if not get_json_from_resp(resp=resp):
```

Comment:
This seems like a lot of unnecessary processing (i.e. we're loading _all_ responses into memory, even potentially very large ones). It also is strange behavior, in that it seems to be catering to another part of our code that isn't clear from just the context of this function.

What's the reasoning here?

Reply:
Also, what if a developer actually wants to return an empty object? This will lead to very confusing behavior - it will replace their empty object with an empty string.

Reply:
Or possibly even more possible, what if a developer wants to return an empty list?

Reply:
Again, matching existing behavior:
https://github.com/plangrid/flask-rebar/blob/master/tests/test_rebar.py#L319

Would make sense to change that, but that would be a change from what we had previously.

Could save the processing here by just checking for other various almost-falsey cases. `b""` or `b"{}"` don't evaluate to false, but do when decoded.